torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -21,7 +21,7 @@
 
 from torchvision.utils import _log_api_usage_once
 
-from ._meta import convert_format_bounding_box, get_spatial_size_image_pil
+from ._meta import clamp_bounding_box, convert_format_bounding_box, get_spatial_size_image_pil
```

Comment:
The corresponding transform is called `ClampBoundingBoxes` (plural). Is that on purpose?